ext/financialacls/financialacls.php

```diff
@@ -85,6 +85,9 @@ function financialacls_civicrm_pre($op, $objectName, $id, &$params) {
  * @link http://wiki.civicrm.org/confluence/display/CRMDOC/hook_civicrm_selectWhereClause
  */
 function financialacls_civicrm_selectWhereClause($entity, &$clauses) {
+  if (CRM_Core_Permission::check([['administer CiviCRM', 'administer CiviCRM Financial Types']])) {
```

Comment:
Super admin is actually 'all CiviCRM permissions and ACLs'

![image](https://github.com/user-attachments/assets/6b9afa70-a861-475f-b08b-89495752fe71)